.github/workflows/trivy-analysis.yml

```diff
@@ -2,22 +2,27 @@ name: Trivy Weekly Scan
 
 on:
   schedule:
-    - cron: '35 6 * * 6'
+    - cron: "35 6 * * 6"
+
+permissions: read-all
 
 jobs:
   scan:
     name: Scan
-    runs-on: ubuntu-20.04
+    runs-on: ubuntu-22.04
+    permissions:
+      security-events: write
     steps:
       - name: Run Trivy vulnerability scanner
```

Comment:
hier nochmal Trivy Scan? Oben in workflows/ci ist trivy ja auch schon enthalten.

Reply:
ist sicher etwas redundant - dieser workflow scannt regelmäßig den latest tag und schreibt die ergebnisse in den security tab. Also wie viele cves in dem container image aktuell drin sind, das trivy  als teil der ci prüft das bei jedem bauen und failed bie high/critical.